androidApp/build.gradle.kts

```diff
@@ -155,9 +155,11 @@ dependencies {
     implementation(libs.compose.material)
     implementation(libs.compose.material.icons.core)
     implementation(libs.compose.material.icons.extended)
-    implementation(libs.compose.navigation)
     implementation(libs.coil.compose)
 
+    implementation(libs.decompose.decompose)
```

Comment:
do you need to `implement` this even if we have it specified it as `api` under shared's build.gradle.kts ?

`api(libs.decompose.decompose)` 🤔

Reply:
Most likely we don't need! Thanks for spotting. Also we can look for unused dependencies in the shared module.